test failure with python 3.5 release candidate build

packaging test's fail with the release candidate build of of python3.5.2:

=================================== FAILURES ===================================
_________________ TestDefaultEnvironment.test_matches_expected _________________

self = <tests.test_markers.TestDefaultEnvironment object at 0x7ff51c8e1278>

    @pytest.mark.skipif(not hasattr(sys, 'implementation'),
                        reason='sys.implementation does not exist')
    def test_matches_expected(self):
        environment = default_environment()
    
        iver = "{0.major}.{0.minor}.{0.micro}".format(
            sys.implementation.version
        )
        if sys.implementation.version.releaselevel != "final":
            iver = "{0}{1}[0]{2}".format(
                iver,
                sys.implementation.version.releaselevel,
                sys.implementation.version.serial,
            )
    
>       assert environment == {
            "implementation_name": sys.implementation.name,
            "implementation_version": iver,
            "os_name": os.name,
            "platform_machine": platform.machine(),
            "platform_release": platform.release(),
            "platform_system": platform.system(),
            "platform_version": platform.version(),
            "python_full_version": platform.python_version(),
            "platform_python_implementation": platform.python_implementation(),
            "python_version": platform.python_version()[:3],
            "sys_platform": sys.platform,
        }
E       assert {'implementat...'x86_64', ...} == {'implementati...'x86_64', ...}
E         Omitting 10 identical items, use -v to show
E         Differing items:
E         {'implementation_version': '3.5.2c1'} != {'implementation_version': '3.5.2candidate[0]1'}
E         Use -v to get the full diff

tests/test_markers.py:134: AssertionError
======== 1 failed, 28376 passed, 2 skipped, 1 xfailed in 41.70 seconds =========

--- a/tests/test_markers.py
+++ b/tests/test_markers.py
@@ -133,7 +133,7 @@ class TestDefaultEnvironment:
             sys.implementation.version
         )
         if sys.implementation.version.releaselevel != "final":
-            iver = "{0}{1}[0]{2}".format(
+            iver = "{0}{1[0]}{2}".format(
                 iver,
                 sys.implementation.version.releaselevel,
                 sys.implementation.version.serial,

